evm_loader/shared-lib/src/lib.rs

```diff
@@ -0,0 +1,297 @@
+            let rpc_client = context::build_rpc_client(&config, slot)?;
+            Ok(serde_json::to_string(
+                &emulate::execute(
+                    rpc_client.as_ref(),
+                    config.evm_loader,
+                    tx_params,
+                    token_mint,
+                    chain_id,
+                    step_limit,
+                    commitment,
+                    &accounts,
+                    &solana_accounts,
+                    trace_call_config,
+                )
+                .await?,
+            )
+            .unwrap())
+        }
+        "get_ether_account_data" => {
+            let param_types::Params {
+                api_options,
+                slot,
+                params: param_types::GetEtherAccountData { ether_address },
+            } = serde_json::from_str(params).map_err(|_| params_to_neon_error(params))?;
+            let config = Arc::new(create_from_api_comnfig(&api_options)?);
+            let rpc_client = context::build_rpc_client(&config, slot)?;
+            Ok(serde_json::to_string(
+                &get_ether_account_data::execute(
+                    rpc_client.as_ref(),
+                    &config.evm_loader,
+                    &ether_address,
+                )
+                .await?,
+            )
+            .unwrap())
+        }
+        "get_neon_elf" => {
+            let param_types::Params {
+                api_options,
+                slot,
+                params: param_types::GetNeonElf { program_location },
+            } = serde_json::from_str(params).map_err(|_| params_to_neon_error(params))?;
+            let config = Arc::new(create_from_api_comnfig(&api_options)?);
+            let rpc_client = context::build_rpc_client(&config, slot)?;
+            let context = context::create(rpc_client, config.clone());
+            Ok(serde_json::to_string(
+                &get_neon_elf::execute(&config, &context, program_location.as_deref()).await?,
+            )
+            .unwrap())
+        }
+        "get_storage_at" => {
+            let param_types::Params {
+                api_options,
+                slot,
+                params:
+                    param_types::GetStorageAt {
+                        ether_address,
+                        index,
+                    },
+            } = serde_json::from_str(params).map_err(|_| params_to_neon_error(params))?;
+            let config = Arc::new(create_from_api_comnfig(&api_options)?);
+            let rpc_client = context::build_rpc_client(&config, slot)?;
+            Ok(serde_json::to_string(
+                &get_storage_at::execute(
+                    rpc_client.as_ref(),
+                    &config.evm_loader,
+                    ether_address,
+                    &index,
+                )
+                .await?,
+            )
+            .unwrap())
+        }
+        "init_environment" => {
```

Comment:
I think, we shouldn't support it in Neon Core API, because it also will give access of unauthorized user to files on server.